invokeai/backend/model_manager/load/model_loaders/generic_diffusers.py

```diff
@@ -28,14 +28,15 @@ class GenericDiffusersLoader(ModelLoader):
 
     def _load_model(
         self,
-        model_path: Path,
-        model_variant: Optional[ModelRepoVariant] = None,
+        config: AnyModelConfig,
         submodel_type: Optional[SubModelType] = None,
     ) -> AnyModel:
+        model_path = Path(config.path)
         model_class = self.get_hf_load_class(model_path)
         if submodel_type is not None:
             raise Exception(f"There are no submodels in models of type {model_class}")
-        variant = model_variant.value if model_variant else None
+        repo_variant = getattr(config, "repo_variant", None)
```

Comment:
There are a couple spots where we could use `isinstance` instead of `getattr`:

```py
repo_variant = config.repo_variant if isinstance(config, DiffusersConfigBase) else None
```

Personally I prefer `isinstance` for clarity but I don't feel too strongly about it.

Reply:
I don't mind either way, but have changed to `isinstance`.